shell/config/features.js

```diff
@@ -1,3 +1,5 @@
 export const ONE_WAY = [
   'token-hashing', 'multi-cluster-management'
 ];
+
+export const HARVESTER_NAME = 'harvester';
```

Comment:
It's scary how often this is still used outside of pkgs. At some point we'll need to address, not now though

Reply:
Yeah, unfortunately I think we've already grabbed the low-hanging fruit and remaining references will take more work to remove.